releasenotes/notes/fix-dynamic-instrumentation-logging-aef13f091e5feed5.yaml

```diff
@@ -0,0 +1,4 @@
+---
+fixes:
+  - |
+    dynamic instrumentation: remove unnecessary log line from application start up
```

Comment:
This note isn't useful for a user. It just calls into question why the removal of this log is a fix.